### ISSUE TITLE
Get weekday name pipe

### ISSUE THREAD

**Issue by breslavsky** [joanllenas/ngx-date-fns#326]
Please check my suggestion for pipe `<div>{{0 | dfnsWeekdayName:weekdayName.short}}</div>`

```javascript
import locale from 'date-fns/esm/locale/ru';

enum WeekdayName {
  full = 'full',
  short = 'short'
}

function getWeekdayName(day: number, view: WeekdayName = WeekdayName.full) {
  const now = new Date();
  const week = eachDayOfInterval({
    start: startOfWeek(now, {locale: locale}),
    end: endOfWeek(now, {locale: locale})
  });
  return format(week[day], view === WeekdayName.short 
    ? 'EEEEEE' : 'EEEE', {locale: locale});
}

console.log(getWeekdayName(0, WeekdayName.short)); // пн
console.log(getWeekdayName(1)); // вторник
```
The pipe will be useful to create calendars as we should write weekdays names based on current locale.

**Comment by JoeFamiliar** [comment]
That will be nice! Wanna it)

**Comment by joanllenas** [comment]
That's nice,
until now this library has been strictly a one to one conversion from date-fns to Angular pipes, but I like the idea of opening a new _utilities_ section to add more value. 
This could be the first of many :)

This weekend I'll work on all pending proposals.
Thanks!

**Comment by breslavsky** [comment]
Thanks a lot for you job and reactive feedback ;-)

We included your library to our open source UI kit for angular. In 1 month we are preparing release and sure, will be a special section about `ngx date fns` how to configure locales.

I will share the link to you soon ;-)

**Comment by joanllenas** [comment]
This is available on v6.1.0.
See: https://github.com/joanllenas/ngx-date-fns/releases/tag/v6.1.0
Closing
